src/modules/organization/home/team/api.ts

```diff
@@ -0,0 +1,38 @@
+import { useQuery } from "@vue/apollo-composable";
+import { GetOrgTeamItemQuery } from "@/apollo/types";
+import gql from "graphql-tag";
+
+//Change later to be dynamic id
+export const useOrgTeamApi = () =>
+  useQuery<GetOrgTeamItemQuery>(gql`
+    query getOrgTeamItem {
+      organization(id: 5) {

```

Comment:
why hard code org?

Reply:
I will handle that in my pr, lets merge.